lms/djangoapps/branding/api.py

```diff
@@ -0,0 +1,214 @@
+"""Edx branding API
+"""
+import logging
+import os
+from django.conf import settings
+from django.utils.translation import ugettext as _
+from django.shortcuts import render_to_response
+from django.core.cache import cache
+from staticfiles.storage import staticfiles_storage
+
+from microsite_configuration import microsite
+from edxmako.shortcuts import marketing_link
+
+log = logging.getLogger("edx.footer")
+
+
+def get_footer_json():
+    """ Get the footer links json
+
+    Returns:
+        Dict of footer links
+    """
+    is_edx_domain = settings.FEATURES.get('IS_EDX_DOMAIN', False)
+    key = "footer/json/{domain}".format(domain="edx" if is_edx_domain else "open_edx")
+    footer_json = cache.get(key)
+    if footer_json:
+        return footer_json
+    site_name = microsite.get_value('SITE_NAME', settings.SITE_NAME)
+
+    context = dict()
```

Comment:
Use literal syntax (i.e., `{}`). It's [faster than using `dict()`](http://stackoverflow.com/questions/30216000/why-is-faster-than-list).